src/commands.py

```diff
@@ -273,3 +274,14 @@ def argos_info(ep):
     "\n-- Excluded:", len(info_data["excluded"]),
     "\n-- UD created:", len(info_data["ud"]),
     "\n-- transactions:", len(info_data["tx"]))
+
+
+def id_pubkey_correspondence(ep, id_pubkey):
+    if check_public_key(id_pubkey):
+        print(id_pubkey, "public key correspond to identity:", get_uid_from_pubkey(ep, id_pubkey))
+    else:
+        pubkey = get_pubkey_from_id(ep, id_pubkey)
+        if pubkey == "No matching identity":
+            print (pubkey)
+        else:
+            print(id_pubkey, "identity correspond to public key:", pubkey)
```

Comment:
Why do you prefer this printing method?
The one used is good for me. I prefer the `.format()` one than yours.